code/public/index.html

```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <!--
       Notice the use of %PUBLIC_URL% in the tags above.
```

Comment:
It would look more clean if you removed the comments in this file, but that's just a suggestion!